src/pages/index.tsx

```diff
@@ -305,152 +320,41 @@ export default function Home() {
-                        alt="X"
-                        width={14}
-                        height={14}
-                        quality={100}
-                      />
-                    </button>
-                  </Transition>
-                </div>
+                  <div className="absolute group-hover:text-orange-600 text-2xl flex justify-center translate-y-32">
+                    <BsChevronCompactDown />
+                  </div>
 
-                <div className="flex flex-col w-[413.33px] h-[508px] items-center relative">
-                  <div className="top-10 relative">
-                    <Image src={cardImage} height={300} width={300} alt="alt" />
-                    <div className="p-2 flex flex-row justify-center space-x-2 ">
-                      {content.colours.map((colour, index) => {
-                        const outlineColour = `outline-${selectedColour}`;
-                        const bgColour = `bg-${colour}`;
-                        return (
-                          <button
-                            onClick={() => {
-                              setSelectedColour(colour);
-                              setCardImage(content.image[index]);
-                            }}
-                            key={index}
-                            className={`${bgColour} h-4 w-4 rounded-full outline-1 outline outline-offset-2 ${
-                              selectedColour !== colour
-                                ? "outline-none"
-                                : outlineColour
-                            }`}
-                          />
-                        );
-                      })}
-                    </div>
-                    <div className="relative flex flex-col items-center flex-wrap top-20 font-[440]">
-                      <div className="text-sm flex flex-row space-x-2 place-content-center">
-                        <div>{content.name}</div>
-                        <div className="text-gray-500 content-center flex">
-                          {content.edition}
-                        </div>
-                      </div>
-                      <div className="text-sm">{content.price}</div>
-                      <button className="relative hover:bg-orange-500 hover:text-white hover:border-orange-500 tracking-wider text-sm font-normal border rounded border-black text- px-5 py-2 top-6 duration-200">
-                        SHOP NOW
-                      </button>
-                    </div>
+                  <div className="group-hover:text-orange-600 text-sm translate-y-4 text-gray-400">
+                    {content.text}
                   </div>
                 </div>
               </div>
             );
           })}
         </div>
+        <div className="flex flex-row flex-wrap justify-center">
+          {walletDataSet.map((content, index) => {
+            return <ProductCard key={index} content={content} index={index} />;
+          })}
```

Comment:
Can probably change from {} to () and omit the return for cleaner implementation

Reply:
updated to () with no returns.